demisto_sdk/commands/validate/validators/BC_validators/BC102_is_context_path_changed.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from itertools import chain
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration]
+
+
+def is_context_path_changed(content_item: Integration) -> bool:
+    """
+    This method checks if an output context path (an integration) is changed.
+    """
+
+    new_outputs_context_path = set(
```

Comment:
will it fail on cases where we add a new command so that old file doesn't have it? especially if we didn't add the command at the bottom of the page?
Also, since we want to return Command A missing a,b,c outputs.. I'm not sure this will work

Reply:
we check that the new integration contains the old one, but could contains more than it